mgr/src/cmds/config_snapshots.cr

```diff
@@ -95,13 +95,16 @@ handler "config-snapshot.restore" do |args|
   STRING
 end
 
-command "config-snapshot.list", "Kadalu Storage Config Snapshots List" do |parser, _args|
+command "config-snapshot.list", "Kadalu Storage Config Snapshots List" do |parser, args|
   parser.banner = "Usage: kadalu config-snapshot list [arguments]"
+  parser.on("--snap-name=snap_name", "Kadalu Snapshots name") do |snap_name|
```

Comment:
`--name`

Reply:
Or you can accept Snap name as positional argument. Like in other commands.

```
kadalu volume list
kadalu volume list POOL
kadalu volume list POOL/VOLUME

kadalu config-snapshot list
kadalu config-snapshot list SNAPNAME
```

Reply:
used pos. args.